ofrak_core/src/ofrak/core/rar.py

```diff
@@ -30,7 +30,10 @@ class RarArchive(GenericBinary, FilesystemRoot):
 
 class RarUnpacker(Unpacker[None]):
     """
-    Unpack RAR archives using the free `unar` tool.
+    Extracts files and directories from RAR compressed archives using the free unar tool. RAR is a
+    proprietary archive format that offers good compression and supports features like split
+    archives and recovery records. Use when analyzing RAR-packaged software, data archives, or
+    firmware bundles. The unpacker handles various RAR versions and compression methods.
```

Comment:
I don't think we have a RAR packer at the moment. Perhaps worth noting in the docstring, since some other docstrings note when their unpackers don't have a corresponding packer.